<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="centralwidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>726</width>
      <height>849</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="3">
      <widget class="QTableWidget" name="tableWidget"/>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Добавить запись</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>Удалить запись</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="loadImageButton">
       <property name="text">
        <string>Загрузить изображение</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="2" column="1">
      <widget class="QListView" name="listView"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="imageLabel">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>Изображение</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item row="3" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="addListButton">
         <property name="text">
          <string>Добавить в список</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeListButton">
         <property name="text">
          <string>Удалить из списка</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sortListButton">
         <property name="text">
          <string>Сортировать список</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="customSortButton">
         <property name="text">
          <string>Особая сортировка</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Сохранить данные</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadButton">
         <property name="text">
          <string>Загрузить данные</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="placeholderText">
        <string>Введите данные в формате: Фамилия,Время,IP (каждая запись с новой строки)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="loadFromTextButton">
       <property name="text">
        <string>Загрузить из текста</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QMenuBar" name="menubar_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>800</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QStatusBar" name="statusbar_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>3</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
